src/chronify/sqlalchemy/functions.py

```diff
@@ -26,8 +29,8 @@ def write_database(df: pd.DataFrame, conn: Connection, schema: TableSchema) -> N
     config = schema.time_config
     if config.needs_utc_conversion(conn.engine.name):
         assert isinstance(config, DatetimeRange)
-        if isinstance(df.timestamp.dtype, DatetimeTZDtype):
+        if isinstance(df[config.time_column].dtype, DatetimeTZDtype):
             df[config.time_column] = df[config.time_column].dt.tz_convert("UTC")
         else:
             df[config.time_column] = df[config.time_column].dt.tz_localize("UTC")
-    pl.DataFrame(df).write_database(schema.name, connection=conn, if_table_exists="append")
+    pl.DataFrame(df).write_database(schema.name, connection=conn, if_table_exists="replace")
```

Comment:
Why replace? It will delete whatever data currently exists? Is there a use case for this? If so, the parameter needs to be passed in. We definitely need append.

Reply:
This is going to be an arg input